### ISSUE TITLE
Share location feature request

### ISSUE THREAD

**Issue by afriedmanGlacier** [ChatSecure/ChatSecure-iOS#698]
Would it be possible to have the ability to share location similar to Google Hangouts and Signal with OMEMO encryption? Not a necessity, but nice to have.

**Comment by chrisballinger** [comment]
Definitely, it would be very useful! We are focused on other things at the
moment, but there is definitely demand for this feature. Ideally we would
also offer a time based location feature similar to iMessage, so you could
asynchronously share your location for an hour, or until the end of the day.
